handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -168,22 +169,23 @@ public JdkSslServerContext(
      * @deprecated use {@link SslContextBuilder}
      */
     @Deprecated
-    public JdkSslServerContext(File trustCertChainFile, TrustManagerFactory trustManagerFactory,
+    public JdkSslServerContext(File trustCertCollectionFile, TrustManagerFactory trustManagerFactory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 11 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)